book/src/drivers.md

```diff
@@ -46,7 +46,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Hedvig](https://documentation.commvault.com/commvault/hedvig/others/pdf/Hedvig_CSI_User_Guide.pdf) | `io.hedvig.csi` | v1.0 | A Container Storage Interface (CSI) Driver for Hedvig | Persistent | Read/Write Multiple Pods | Yes | Snapshot
 [Hetzner Cloud Volumes CSI](https://github.com/hetznercloud/csi-driver) | `csi.hetzner.cloud` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Hetzner Cloud Volumes | Persistent | Read/Write Single Pod | Yes | Raw Block, Expansion
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
-[HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v1.0, v1.1, v1.2 | A [multi-vendor](https://scod.hpedev.io/csi_driver) Container Storage Interface (CSI) driver. Supports [HPE Nimble Storage](https://hpe.com/storage/nimble), [HPE Primera](https://hpe.com/storage/primera) and [HPE 3PAR](https://hpe.com/storage/3par) | Persistent and Ephemeral | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion and Cloning
```

Comment:
What does "multi-vendor" mean?  Does it mean this CSI driver supports non-HPE storage?

Reply:
@xing-yang yes it does. The HPE CSI Driver architecture allows any block storage (iSCSI/FC) vendor to implement something we call a [CSP](https://github.com/hpe-storage/container-storage-provider), which is an API gateway to their backend. There's one 3rd party implementation in the works that we know of but it has not been released yet.

Reply:
Sure. That would be similar to Ember CSI driver in the long run.  I think if you want to list the 3 HPE drivers here, then "multi-vendor" should be removed as it is confusing.  The alternative would be to remove the reference of the 3 HPE drivers and keep "multi-vendor".  In the later case, this page "https://github.com/hpe-storage/csi-driver" should talk about how this could support multi-vendor.  It is still confusing though as it is under "hpe-storage".  So I think it is better to remove "multi-vendor" here, but you can still explain about it in your own driver doc.

Reply:
Thanks. Reworded multi-vendor to multi-platform.

Reply:
Sounds good.  Thanks.